import "common.proto";
message LoginInfo
{
    required int32 login_day = 1; // 登录天数 1-7
    required int32 state = 2; // -1 未达成 0 已达成未领取 1 已领取
}

//  login gift info 825
message InitLoginGiftResponse{
	repeated LoginInfo cumulative_day = 1;  //累积登录
	repeated LoginInfo continuous_day = 2;  //连续登录
}

// get login gift 826
message GetLoginGiftRequest{
	required int32 activity_id = 1;
	required int32 activity_type = 2;
}

message GetLoginGiftResponse{
	required bool result = 1;
	optional int32 result_no = 2;
    optional GameResourcesResponse gain = 3;
}
